src/generated/resources/data/tconstruct/recipes/tools/armor/plate_boots.json

```diff
@@ -1,5 +1,6 @@
 {
   "type": "tconstruct:tool_building",
   "result": "tconstruct:plate_boots",
-  "result_count": 1
-}
\ No newline at end of file
+  "result_count": 1,
+  "slot_id": "tconstruct:plate_armor"
```

Comment:
Call it `layout` or `slot_layout`, `slot_id` is unclear and 90% of recipes are IDs